data/Makefile.am

```diff
@@ -28,16 +36,23 @@ else
 endif
 
 appletdir        = $(datadir)/mate-panel/applets
-applet_DATA      = org.mate.applets.MateNotificationApplet.mate-panel-applet
-applet_in_files  = $(applet_DATA:.mate-panel-applet=.desktop.in)
+applet_DATA = $(applet_in_files:.mate-panel-applet.desktop.in=.mate-panel-applet)
+applet_in_files  = org.mate.applets.MateNotificationApplet.desktop.in
+
+$(applet_in_files): $(applet_in_files).in
+	$(AM_V_GEN)sed \
+		-e "s|\@APPLET_LOCATION\@|$(APPLET_LOCATION)|" \
+		-e "s|\@APPLET_IN_PROCESS\@|$(APPLET_IN_PROCESS)|" \
+		$< > $@
 
 $(applet_DATA): $(applet_in_files)
 if USE_NLS
 	$(AM_V_GEN) $(MSGFMT) --desktop --keyword=Name --keyword=Description --template $< -d $(top_srcdir)/po -o $@
 else
-	$(AM_V_GEN) cp -f $< $@
+	$(AM_V_GEN) cp $< $@
 endif
```

Comment:
I don't see `if ENABLE_IN_PROCESS/else/endif` for this part. Maybe this is the problem?

Reply:
By defining the applet location in configure.ac, I was able to avoid any changes whatsoever to data/Makefile.am

Since ENABLE_IN_PROCESS was defined in configure.ac it was passed directly to building 
org.mate.applets.MateNotificationApplet.desktop.in.
so all I had to do was also define APPLET_LOCATION there. Took some fiddling but I got it